include/crm/cib/cib_types.h

```diff
@@ -80,6 +80,10 @@ enum cib_call_options {
     cib_scope_local     = (1 << 8),
 
     cib_dryrun          = (1 << 9),
+
+    //! Add request to the client's CIB transaction instead of processing
+    cib_transaction     = (1 << 10),
```

Comment:
This is cib_extend_transaction in the rest of the commits

Reply:
Thanks yeah, I fixed it locally after I rebased and pushed